src/MSBuild/MSBuild.csproj

```diff
@@ -187,6 +187,8 @@
     </Compile>
     <Compile Include="NodeEndpointOutOfProcTaskHost.cs" />
     <Compile Include="LogMessagePacket.cs" />
+    <Compile Include="NodeEndpointRarTaskHost.cs" />
+    <Compile Include="RarServiceTaskHostNode .cs" />
```

Comment:
Extra space.

Reply:
interesting... i haven't modified iot by hand + how come it compiles?

Reply:
fixed anyway

Reply:
>how come it compiles?

that's the question I asked myself too..